docs/reset-techpass-mfa-for-new-device.md

```diff
@@ -10,37 +10,37 @@ If you have a new mobile device, you'll need to set it up to work with multi-fac
 
 ?> If you are a public officer and trying to access your TechPass account from a GMD, you will be prompted to authenticate your SG GovtM365 account before authenticating TechPass sign-in.
 
-<kbd>![](../assets/images/reset-up-techpass-mfa-vendor/security-info-menu.png)</kbd>
+<kbd>![](assets/images/reset-techpass-mfa-vendor/security-info-menu.png)</kbd>
 
 3.  On the **Security info** page, click **Delete** next to the Authenticator method linked to your old mobile device.
 
-<kbd>![delete-auth-method](../assets/images/reset-up-techpass-mfa-vendor/delete-auth-app-for-old-device.png)
+<kbd>![delete-auth-method](assets/images/reset-techpass-mfa-vendor/delete-auth-app-for-old-device.png)
 
 4. Click **OK** to confirm the deletion.
 
 5. Open the authenticator app on your old mobile device, delete your TechPass account from the authenticator app.
 
 6. On the **Security info** page, click **+ Add method**, select **Authenticator app** and click **Add**.
 
-<kbd>![add-auth-method](../assets/images/reset-techpass-mfa-vendor/add-method.png)
+<kbd>![add-auth-method](/assets/images/reset-techpass-mfa-vendor/add-method.png)
```

Comment:
this has additional / in front compared to other paths. does it matter?

Reply:
looks like the local preview is fine but still i don't want to take chances , will remove this Thank you !